src/context/UserContext.jsx

```diff
@@ -143,14 +142,6 @@ const UserProvider = ({children}) => {
                         toast.error(
                             <div>
                                 <span>Session has expired, Please Sign-in</span>
-                                <Button
-                                    className="google-calendar-btn col-lg-3 mt-3"
-                                    variant="secondary"
-                                    size="lg"
-                                    onClick={clearStateAndRedirect}
-                                >
-                                    Go to Home
-                                </Button>
```

Comment:
@Danielsio Have you removed that part becouse you couldn't call `clearStateAndRedirect` for the Context (can't navigate)?

Reply:
Correct